artifacts/LogonScript.ps1

```diff
@@ -19,611 +19,73 @@ $DisableArcAgentAutoUpgrade = if ($env:disableArcAgentAutoUpgrade) { $env:disabl
-    Write-Host "Supported Locations : $arcLocations"
-    exit -1
-}
-
-# Validate az cli version.
-$azVersion = (az version)[1].Split(":")[1].Split('"')[1]
-$azMinRequiredVersion = "2.64.0"
-if ($azVersion -lt $azMinRequiredVersion){
-    Write-Host "Installed Azure CLI version $azVersion is older than $azMinRequiredVersion. Please upgrade Azure CLI and retry." -ForegroundColor Red
-    exit -1
-}
-
-# Ensure logged into Azure
-$azureLogin = az account show
-if ( $null -eq $azureLogin){
-    Write-Host "Please login to azure via `az login` and retry." -ForegroundColor Red
-    exit -1
-}
-
-# Ensure `connectedk8s` az cli extension is installed and up to date.
-$errOut = $($retVal = & {az extension add --upgrade --name connectedk8s -y --allow-preview false}) 2>&1
-if ($LASTEXITCODE -ne 0)
-{
-    throw "Error upgrading extension connecktedk8s : $errOut"
-}
-
-$installDir = $((Get-Location).Path)
-$productName = "AKS Edge Essentials - K3s"
-$networkplugin = "flannel"
-if ($UseK8s) {
-    $productName ="AKS Edge Essentials - K8s"
-    $networkplugin = "calico"
-    # Setting URL to empty string, so K8s msi will be selected
-    $productUrl = ""
-}
-
-# Here string for the json content
-$aideuserConfig = @"
-{
-    "SchemaVersion": "1.1",
-    "Version": "1.0",
-    "AksEdgeProduct": "$productName",
-    "AksEdgeProductUrl": "$productUrl",
-    "Azure": {
-        "SubscriptionName": "",
-        "SubscriptionId": "$SubscriptionId",
-        "TenantId": "$TenantId",
-        "ResourceGroupName": "$ResourceGroupName",
-        "ServicePrincipalName": "aksedge-sp",
-        "Location": "$Location",
-        "CustomLocationOID":"$CustomLocationOid",
-        "EnableWorkloadIdentity": true,
-        "EnableKeyManagement": true,
-        "GatewayResourceId": "",
-        "Auth":{
-            "ServicePrincipalId":"",
-            "Password":""
-        }
-    },
-    "AksEdgeConfigFile": "aksedge-config.json"
-}
-"@
-$aksedgeConfig = @"
-{
-    "SchemaVersion": "1.14",
-    "Version": "1.0",
-    "DeploymentType": "SingleMachineCluster",
-    "Init": {
-        "ServiceIPRangeSize": 10
-    },
-    "Network": {
-        "NetworkPlugin": "$networkplugin",
-        "InternetDisabled": false,
-        "Proxy": {
-            "Http": null,
-            "Https": null,
-            "No": null
-        }
-    },
-    "User": {
-        "AcceptEula": true,
-        "AcceptOptionalTelemetry": true
-    },
-    "Machines": [
-        {
-            "LinuxNode": {
-                "CpuCount": "$CpuCoreCount",
-                "MemoryInMB": $VMMemory,
-                "DataSizeInGB": 40,
-                "LogSizeInGB": 4
-            }
-        }
-    ]
-}
-"@
+# download public script and config json tempaltes
+$scriptUrl = "https://raw.githubusercontent.com/jagadishmurugan/AKS-Edge/refs/heads/users/jagamu/pvt-accept-input-from-config-file/tools/scripts/AksEdgeQuickStart/AksEdgeQuickStartForAio.ps1"
```

Comment:
Shouldn't this be the public location of the script ? and not Jagadish's branch ?

Reply:
updated to main